scripts/rabbitmq-env

```diff
@@ -87,8 +87,14 @@ DEFAULT_SCHEDULER_BIND_TYPE="db"
 DEFAULT_DISTRIBUTION_BUFFER_SIZE=128000
 [ "x" = "x$RABBITMQ_DISTRIBUTION_BUFFER_SIZE" ] && RABBITMQ_DISTRIBUTION_BUFFER_SIZE=${DEFAULT_DISTRIBUTION_BUFFER_SIZE}
 
+DEFAULT_MAX_NUMBER_OF_PROCESSES=1048576
+[ "x" = "x$RABBITMQ_MAX_NUMBER_OF_PROCESSES" ] && RABBITMQ_MAX_NUMBER_OF_PROCESSES=="${DEFAULT_MAX_NUMBER_OF_PROCESSES}"
```

Comment:
`==` seems to be bash-specific and is not available in zsh.

Reply:
My bad, definitely a typo.